.circleci/config.yml

```diff
@@ -203,22 +201,27 @@ jobs:
 
       - copy-api-tests-and-utils-to-subtester
 
-      - activate-license
+      - unity/prepare-env:
+          project-path: Subtester
 
       - resolve-dependencies:
           project: Subtester
           target: Android
 
-      - perform-build:
-          project: Subtester
-          target: Android
+      - unity/build:
+          project-path: Subtester
+          build-target: Android
+          build-name: Android
+          build-method: BuildCommand.PerformBuild
+          custom-parameters: |
+            -customBuildOptions AcceptExternalModificationsToPlayer
 
       - run:
           name: Verify APK exists
-          command: test -f Subtester/buildAndroid.apk
+          command: test -f Subtester/Builds/Android/Android.apk
 
       - store_artifacts:
-          path: Subtester/buildAndroid.apk
+          path: Subtester/Builds/Android/Android.apk
 
   build-subtester-ios:
```

Comment:
This works so I didn't change it to use the orb and it uses our build command. It's basically the same anyways